lib/matplotlib/__init__.py

```diff
@@ -215,7 +222,76 @@ def _is_writable_dir(p):
     """
     return os.access(p, os.W_OK) and os.path.isdir(p)
 
+_verbose_msg = ("--verbose-LEVEL and the Verbose class are deprecated. "
+                "This functionality is now provided by the standard "
+                "python logging library.  To get more (or less) logging "
+                " output:\n"
+                "import logging\n "
+                "logger = logging.getLogger('matplotlib')."
+                "logger.set_level(logging.INFO)")
```

Comment:
Looks like you need to add newlines in this line and the one above.

Reply:
Also remove the dot at the end of the previous line -- or just squeeze the whole thing into `logging.getLogger('matplotlib').set_level(logging.INFO)`
Also remove the space in `" output:"`.

TBH I would consider using a triple quoted string here:
```
# note the backslash in the first line!
_verbose_msg = """\
--verbose-LEVEL and the Verbose class etc.
import logging
logging.getLogger('matplotlib').set_level(logging.INFO)"""
```